### PR TITLE
Extend config schema to support PDO options.

### PR DESCRIPTION
**Pull request by scottmckenzie**
Extending the schema with this option allows me to connect to MySQL over a TLS connection.

### CODE REVIEW

File: config/api_sample.php [hunk 1]
```diff
@@ -24,6 +24,9 @@
         // When using unix socket to connect to the database the host attribute should be removed
         // 'socket' => '/var/lib/mysql/mysql.sock',
         'socket' => '',
+        'driver_options' => [
```
Comment (WoLfulus) [inline config/api_sample.php]: Please comment these lines as they might cause issues with most people that are not using ssl

File: src/core/Directus/Config/Schema/Schema.php [hunk 2]
```diff
@@ -38,6 +38,7 @@ public static function get() {
                 new Value('engine', Types::STRING, 'InnoDB'),
                 new Value('charset', Types::STRING, 'utf8mb4'),
                 new Value('socket', Types::STRING, ''),
+                new Value('driver_options', Types::ARRAY, []),
```
Comment (WoLfulus) [inline src/core/Directus/Config/Schema/Schema.php]: As this didn't exist before, please use "?" to completely remove the value from the final object if it's not set. I'm not sure if make [] the default value can cause problems.

File: config/api_sample.php (hunk 1 above)
Comment (scottmckenzie) [reply config/api_sample.php]: done

File: src/core/Directus/Config/Schema/Schema.php (hunk 2 above)
Comment (scottmckenzie) [reply src/core/Directus/Config/Schema/Schema.php]: done

**Pull request merged**